miner/worker.go

```diff
@@ -451,6 +453,29 @@ func (w *worker) mainLoop() {
 					w.commit(uncles, nil, true, start)
 				}
 			}
+		// Read from the sync service and mine single txs
```

Comment:
We should never receive more than 1 tx here and we should never receive a block with more than 1 tx. I don't want to panic on those cases in case somebody figures out how to include more than one tx in a block. The only way to do so would be to connect to the sequencer on the p2p network (which is not exposed via firewall) because the codepath that we use to produce blocks only produces blocks with single txs

Reply:
Should be sure to run with `--nodiscover` as well
https://github.com/ethereum-optimism/go-ethereum/blob/fc6b5319b3df033206d7488414c278db2054dcb0/cmd/utils/flags.go#L670